client/sites-dashboard-v2/sites-dashboard-header.tsx

```diff
@@ -56,9 +52,47 @@ export const PageBodyBottomContainer = styled.div( {
 	},
 } );
 
+const responsiveButtonStyles = {
+	alignItems: 'center',
+	fontSize: '12px',
```

Comment:
Can we use this variable: https://github.com/Automattic/wp-calypso/blob/3f20d3141ffdfc929b1d4e709aca02eb673f495a/packages/typography/styles/variables.scss#L46 in emotion?

Reply:
Good question! I couldn't find any instance in Calypso that does this. I'm more inclined to use either SCSS or CSS-in-JS, but mixing both seems odd. Perhaps we can address this in the future and use SCSS only since it seems better supported by core and Calypso.